### ISSUE TITLE
A4046 - CE-2-AI214: user interface control missing a programmatically determined name

### ISSUE THREAD

**Issue by RolandCampbell** [RolandCampbell/ForgetMeNot1#11]
*Common Element*: Offers bar\\ *Issue link*: [AI214|http://aqa.usablenet.com/training-and-demo/roa/1642259846e_a92df74362b1807cbeeadc9ac5b9ee97/A4046/T10232/step1/details/view/AI214]\\ *Description*: user interface control missing a programmatically determined name\\ *WCAG Success Criterion*: 4.1.2\\ *Element*: select\\ *Severity*: high\\ *Complexity*: easy
